types/cart-discount/updates/CartDiscountSetDescriptionAction.raml

```diff
@@ -1,11 +1,12 @@
 #%RAML 1.0 DataType
 (package): CartDiscount
 (docs-uri): https://docs.commercetools.com/http-api-projects-cartDiscounts.html#set-description
-
 type: CartDiscountUpdateAction
 displayName: CartDiscountSetDescriptionAction
 discriminatorValue: setDescription
 (postman-example): !include ../../../examples/CartDiscount/CartDiscountSetDescriptionAction.json
 properties:
   description?:
     type: LocalizedString
+    description: If the `description` parameter is not included, the field will be
+      emptied.
```

Comment:
This description is multi line, shouldn't it then start with `|`?

Reply:
Parser checks passed so I guess it ok to have it without | .